synchro/__init__.py

```diff
@@ -483,6 +484,7 @@ class Collection(Synchro):
     aggregate = WrapOutgoing()
     aggregate_raw_batches = WrapOutgoing()
     list_indexes = WrapOutgoing()
+    list_search_indexes = WrapOutgoing()
```

Comment:
We should add the other search methods too.

Reply:
Done